packages/client3/components/design-system/tabs/tabs.tsx

```diff
@@ -13,10 +13,10 @@ export function TabButton({ children }: TabProps) {
       {({ selected }) => (
         <button
           className={clsx(
-            "mr-1 -mb-px rounded-t-md border py-4 px-5 text-sm font-medium",
+            "mr-5 border-transparent",
```

Comment:
Oooh cool, is `border-transparent` to prevent the box from jumping around by `1px`?